src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
+   * @returns {*}
+   */
+  function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
```

Comment:
Another way to go would be to piggy back on the `md-autocomplete-snap` attribute and give it an optional value `md-autocomplete-snap="width"`. This would avoid adding another attribute and then we could only have to walk up the DOM once.